tutorial: hide output of y array; mention default

waldyrious opened the pull request:
I'm not sure this is compatible with @ChrisRackauckas's original intent, given the contents of the [Plotting backends](https://juliaplots.github.io/tutorial/#plotting-backends) section, but I think that even if we choose not to mention Plotly at this point, we should point out that the plot is opened in the browser.

ChrisRackauckas commented:
Hmm, but it doesn't always go to the browser though? That's the tricky part. But yes, something should be mentioned immediately.

mkborregaard commented on docs/tutorial.md:
```diff
@@ -14,8 +14,8 @@ numbers. For example:
-x = 1:10; y = rand(10) # These are the plotting data
-plot(x,y)
+x = 1:10; y = rand(10); # These are the plotting data
+plot(x,y) # uses Plotly by default, and opens the plot it in the browser
```
This is confusing - it only uses Plotly as a default if no backend pages are installed.

waldyrious replied:
Indeed. In face, as I mentioned in the opening comment, I'm not even sure mentioning backends at this point is consistent with the intended information flow for the document, so I'm fine removing this mention of Plotly.

waldyrious commented:
> Hmm, but it doesn't always go to the browser though? That's the tricky part. But yes, something should be mentioned immediately.

I'm open to any suggestions :)

mkborregaard commented:
So are we (sort of) :-)

waldyrious commented:
I'm not very familiar with the behavior, so any help would be appreciated before I can improve the PR. What are the possibilities? I.e. if the user hasn't installed any backend explicitly, what will happen? Is there any backend that will take precedence over Plotly even if not explicitly activated? Are there cases where even with no other backends available, Plotly will produce the image somehow other than opening a page in the browser? And for each of these cases, how likely are them? I mean, it's certainly desirable to be exhaustive in reference documentation, but probably not in a tutorial (especially not in the first lines!).

ChrisRackauckas commented:
>I.e. if the user hasn't installed any backend explicitly, what will happen? Is there any backend that will take precedence over Plotly even if not explicitly activated? Are there cases where even with no other backends available, Plotly will produce the image somehow other than opening a page in the browser?

If no backend is installed, then it will used Plotly if it installed correctly. But if any other backend was previously installed (which includes things like PyPlot), then it will default to those. If Plotly did not install correctly and no backend is installed, then it will default to UnicodePlots backend.

I'm not sure how to condense that though.

mkborregaard commented:
The behaviour also differs between REPL, Juno and Jupyter.

mkborregaard commented:
Maybe just "the plot is displayed in a plot pane, a stand-alone window or the browser, depending on the environment and backend (see below)"?

waldyrious commented:
Thanks for the clarifications. @mkborregaard's suggestion sounds good to me.

waldyrious commented:
@ChrisRackauckas, @mkborregaard -- apologies for the delayed response. I've rebased the branch and updated it with the changes discussed above. Could you reopen the PR?

mkborregaard commented:
I can't reopen - could you just open a new PR?

waldyrious commented:
Sure.
